tests/test_rio_warp.py

```diff
@@ -620,3 +620,27 @@ def test_unrotate(runner, tmp_path):
     with rasterio.open(outputname) as src:
         assert src.transform.b == 0.0
         assert src.transform.d == 0.0
+
+
+def test_coordinate_operation(runner, tmp_path):
```

Comment:
It may be a good idea to parametrize this and pass in both `--transformer-option` and `--to` into the test.